src/jit/gentree.cpp

```diff
@@ -15641,7 +15651,7 @@ bool GenTree::isContained() const
     // if it's contained it better have a user
     if (isMarkedContained)
     {
-        assert((gtNext != nullptr) || OperIsLocal());
+        assert(gtNext != nullptr);
```

Comment:
FWIW, this is a relatively weak condition. We should really add a phase to `CheckLIR` that verifies that any node that is marked contained has a user. If that seems too heavyweight, we could instead change this assert to `assert(!isMarkedContained || !IsUnusedValue())`.

Reply:
Thanks, will be fixed in the next PRs.